GitUpKit/Interface/GIGraph.m

```diff
@@ -523,13 +496,13 @@ - (void)_computeNodePositions {
 
     layer.y = layer.index;
   }
-  _size = CGSizeMake(maxX, CFArrayGetCount(_layers) - 1);
+  _size = CGSizeMake(maxX, self.layers.count - 1);
 }
 
 #if __GI_HAS_APPKIT__
 
 - (void)_computeNodeAndLineColors {
-  NSArray* colors = [NSColor.gitUpGraphAlternatingBranchColors retain];
+  NSArray* colors = [NSColor.gitUpGraphAlternatingBranchColors copy];
```

Comment:
Passing Obj-C objects into CF contexts always stretches my brain a bit. I'll do some checking to make sure we aren't leaking or otherwise releasing the object in correctly here.